workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
Automake uses the variable name, so `icondir` tells where to put the files in variable `icon_DATA` (or in the case here `dist_icon_DATA` - not sure what the `dist_` adds here since the stuff in `_DATA` should be distributed anyway).

Probably the best way to install multiple sizes is to do [like Geany does](https://github.com/geany/geany/tree/master/icons) and put them into separate subdirs for each size, and in the Makefile.am in each subdir, do like you did here.